README.md

```diff
@@ -90,4 +90,37 @@ spec:
       - name: multiclusterhub-operator-pull-secret
 ```
 
+
+Metrics
+==================
+
+Search-v2-api also monitors and exports various metrics to Prometheus. Below is what search api currently monitors.
+
+Histograms:
+
+* `search_http_duration_seconds` - Latency of of HTTP requests in seconds.
+* `search_dbquery_duration_seconds` - Latency of DB requests in seconds.
+   * `resolveItemsFunc`- time it takes to resolve search query
+   * `resolveAutoComplete` - time it takes to resolve search complete
+   * `resolveCountFunc` - time it takes to resolve count
+* `search_dbquery_build_duration_seconds` - Latency of DB query build in seconds.
+   * `buildRbacClause` - time takes to build RBAC clause
+   * `buildRelationshipQuery` - time is takes to build relationship query
+
+
+Counters:
+
+* `search_http_total` - Total number HTTP requests.
```

Comment:
Is it possible to get the count from the histogram `search_http_duration_seconds`?

Reply:
Yes! Histogram creates a counter for search_http_duration_seconds automatically. So, it's already featured in the response

Reply:
example in last line below:
```
# HELP search_dbquery_duration_seconds Latency of DB requests in seconds.
# TYPE search_dbquery_duration_seconds histogram
search_dbquery_duration_seconds_bucket{query="buildRelationshipQuery",le="0.005"} 1
.....
search_dbquery_duration_seconds_bucket{query="buildRelationshipQuery",le="+Inf"} 1
search_dbquery_duration_seconds_sum{query="buildRelationshipQuery"} 0.003396569
search_dbquery_duration_seconds_count{query="buildRelationshipQuery"} 1
```

Reply:
Confirmed that the total count is available in the histogram.
https://prometheus.io/docs/concepts/metric_types/#histogram